src/style.css

```
@@ -1,3 +1,139 @@
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  --button-color: rgba(0, 172, 220, 1);
+  --main-text-color: rgba(6, 11, 53, 1);
+  --secondary-text-color: rgba(97, 96, 112, 1);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  padding: 16px;
+  flex-direction: column;
+  width: 200px;
+
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding-top: 32px;
+  align-items: center;
+  color: var(--main-text-color);
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+}
+
+.card__item-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  margin: 0;
+  margin-top: 40px;
+}
+
+.card__code {
+  margin: 0;
+  margin-top: 4px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: var(--secondary-text-color);
+  align-self: flex-start;
+}
+
+.card__reviews {
+  margin: 0;
+  margin-top: 16px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.card__starbar {
+  display: flex;
+  justify-self: left;
+}
+
+.card__starbar--0 :nth-child(-n+0),
+.card__starbar--1 :nth-child(-n+1),
+.card__starbar--2 :nth-child(-n+2),
+.card__starbar--3 :nth-child(-n+3),
+.card__starbar--4 :nth-child(-n+4),
+.card__starbar--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__star:last-child {
+  margin-right: 0;
+}
+
+.card__reviews-text {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 16px;
+  margin: 0;
+}
+
+.card__price {
+  width: 100%;
+  display: flex;
+  margin-top: 24px;
+  justify-content: space-between;
+}
+
+.card__price-text {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--secondary-text-color);
+}
+
+.card__price-numb {
+  margin: 0;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__buy {
+  margin-top: 16px;
+  width: 100%;
+  height: 40px;
+  padding: 0;
+  color: white;
```

it's better to use hex values instead of reserved words
```suggestion
  color: #ffffff;
```